justfile

```diff
@@ -45,6 +45,10 @@ check-crates-msrv:
 check-docs:
 	@bash contrib/scripts/check-docs.sh
 
+# Build nostr examples
+examples:
+	cargo build --examples --release
+
```

Comment:
I think that who want to run an example, build and run only the one is interested in, so this command IMO is not really needed.